### DISCUSSION TITLE
Possible to provision always encrypted tables?

### DISCUSSION THREAD

**Discussion by ChrisStain87** [Mimetis/Dotmim.Sync#693] [Q&A]
Hello!

Is it possible to sync tables with always encrypted columns? I get an "Operand type clash" error which is logical if the dbtype isn't set correctly, are there any possibilites with dotmim to make it work?

**Comment by Mimetis** [comment]
I don't think it's possible.
Honestly I never tested it.
Maybe if you provide a simple repro, I could make some quick search on it
